workflow-bot-app/src/test/java/com/symphony/bdk/workflow/utils/UtilityFunctionsMapperTest.java

```diff
@@ -0,0 +1,52 @@
+package com.symphony.bdk.workflow.utils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.symphony.bdk.workflow.engine.camunda.UtilityFunctionsMapper;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+@SuppressWarnings("unchecked")
+class UtilityFunctionsMapperTest {
+
+  @Test
+  void testString() {
```

Comment:
maybe add json as part of the method's name? (because we have other methods in this class too)